ecosystem/helper/src/node/bundler/vite/mergeViteConfig.ts

```diff
@@ -0,0 +1,188 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
This file is absolutely needed, as some developers will choose to achieve config merging by import this function from 'vite', however, no matter how slim vite it is, it's still not a small package, so extracting `mergeViteConfig` and provide it standalone should benifit those who choose `bundler-webpack`